build.sbt

```diff
@@ -123,4 +123,8 @@ lazy val examples = Project("blaze-examples",file("examples"))
 
 /* Helper Functions */
 
-addCommandAlias("validate", ";scalafmtCheckAll ;javafmtCheckAll ;test ;unusedCompileDependenciesTest ;mimaReportBinaryIssues")
+// use it in the local development process
+addCommandAlias("validate", ";scalafmtCheckAll ;javafmtCheckAll ;+test:compile ;test ;unusedCompileDependenciesTest ;mimaReportBinaryIssues")
+
+// use it in the CI pipeline
```

Comment:
What is the difference between this and `validate`?  Just `test` vs. `test:compile; test`?

Reply:
Yep, here `+test:compile` to cross-compile module+tests for all Scala versions and `test` for running tests only for a single version. On CI pipeline's it's not needed because there are jobs that run for every Scala version separately.

Reply:
I missed the `+` nuance.  What you're doing makes sense to me.

Reply:
Thanks!